### ISSUE TITLE
Improve error messages in the UI

### ISSUE THREAD

**Issue by josecelano** [torrust/torrust-index#81]
If you try to upload a new torrent with the same name you get a generic error:

![image](https://user-images.githubusercontent.com/58816/176865311-2526d935-72ef-432a-8e1d-3f179fc08c2c.png)

**Comment by josecelano** [comment]
Besides, when you try to ban an admin it says "user not found" instead of "admins cannot be deleted"

**Comment by mickvandijke** [comment]
- [x] https://github.com/torrust/torrust-index-backend/issues/41
- [ ] https://github.com/torrust/torrust-index-backend/issues/42

**Comment by josecelano** [comment]
The error for the duplicate torrent title has been changed in the new frontend version:

![image](https://github.com/torrust/torrust-index/assets/58816/72b4e922-4b9b-4996-8184-077a65d2e5dc)

Regarding the error message when you try to ban the admin, the new version of the frontend does not allow banning users. We lost that feature when we migrated the Vue3. [I've created a new issue for that](https://github.com/torrust/torrust-index/issues/91).
